sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -1636,6 +1657,96 @@ public void outputWithTimestamp(OutputT output, Instant timestamp) {
     }
   }
 
+  private class SizedRestrictionWindowObservingProcessBundleContext
```

Comment:
Class comment that this context outputs `KV<KV<Element, Restriction>, KV<Size, WatermarkEsimtatorStateT>>`